tests/roles/stop_openstack_services/tasks/main.yaml

```diff
@@ -6,6 +6,7 @@
       CONTROLLER2_SSH="{{ controller2_ssh }}"
       CONTROLLER3_SSH="{{ controller3_ssh }}"
 
+# TODO(bogdando): split Compute services on dataplane after the multi-node/multi-cells adoption supported
```

Comment:
Indeed, the below works thanks to being Standalone where nova-compute is collocated with the controller services, but the snippet in the docs and the one below ignores compute nodes. We probably need to introduce COMPUTE${i}_SSH variables. The OVN adoption procedure also SSH's into the compute node and should be eventually able to work with multiple computes, so this issue is not specific to Nova currently. Not a blocker.

Reply:
stoping the nova/libvirt service is somthing i think can and should be done in the relevant edpm roles yes

just be aware that we also run nova-compute with ironic on the controller and in that case the controller will not be an edpm compute as that nova-compute will be moving to openshift.

Reply:
At that time the compute services are pointed at the old RabbitMQ. We aren't stopping Rabbit right now but i wonder if we should. Also we are stopping the Nova controller services, so even if the compute services do make RPC calls and Rabbit is running, there's no service running that could alter DBs i think.